scripts/test/shared.py

```diff
@@ -371,7 +371,7 @@ def fail_if_not_contained(actual, expected):
 def fail_if_not_identical_to_file(actual, expected_file):
     binary = expected_file.endswith(".wasm") or type(actual) == bytes
     with open(expected_file, 'rb' if binary else 'r') as f:
-        fail_if_not_identical(actual, f.read(), fromfile=expected_file)
```

Comment:
I think this wrong.  I'm seeing this failure on a bot which is backwards:

```
incorrect output, diff:

--- D:\a\binaryen\binaryen\test\passes\dwarfdump.bin.txt
+++ actual
@@ -44,7 +44,7 @@
                 DW_AT_high_pc [DW_FORM_data4]	(0x00000002)
                 DW_AT_linkage_name [DW_FORM_strp]	( .debug_str[0x000000c8] = "_Z3foov")
                 DW_AT_name [DW_FORM_strp]	( .debug_str[0x000000d0] = "foo")
-                DW_AT_decl_file [DW_FORM_data1]	("/usr/local/google/home/azakai/Dev/emscripten\a.cpp")
+                DW_AT_decl_file [DW_FORM_data1]	("/usr/local/google/home/azakai/Dev/emscripten/a.cpp")
                 DW_AT_decl_line [DW_FORM_data1]	(3)
                 DW_AT_external [DW_FORM_flag_present]	(true)
```

In this case the actual has the windows backslash .. so and should be preceded by the "+"

Reply:
Ah yes, I think you're right - I was confused before, I guess...

Fix in #2649